maindialog.py

```diff
@@ -153,13 +157,11 @@ def toggleOptions(self):
                         treeOption.setDisabled(False)
 
     def previewMap(self):
-        if not webkit_available:
```

Comment:
This was added to handle Xenial installs which had no QWebKit - probably needs to be tested in that environment